frontend/src/components/AddDatasetDrawer/AddDatasetDrawer.tsx

```diff
@@ -15,8 +15,8 @@ export let AddDatasetDrawer = withStore('selectedDataset')(({ }) =>
       To analyze the accessibility of your own set of providers, facilities, or social services, you
       will need to upload two separate CSV files:
       <ul>
-        <li>list of service areas (County and/or ZIP columns).</li>
-        <li>list of addresses for providers or services.</li>
+        <li>List of service areas (County and/or ZIPCode columns).</li>
```

Comment:
`ZIPCode` is not a currently accepted alias.

Reply:
False, but Zip is simpler :)

Reply:
`  { aliases: ['ZipCode', 'Zip Code', 'Zip'] }`